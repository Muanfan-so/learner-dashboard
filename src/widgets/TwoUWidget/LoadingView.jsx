import React from 'react';
import { Skeleton } from '@edx/paragon';

export const LoadingView = () => (
  <div className="loading-callouts-container">
    <h2 className="skeleton-title-width"><Skeleton /></h2>
    <p><Skeleton /></p>
  </div>
);

export default LoadingView;
