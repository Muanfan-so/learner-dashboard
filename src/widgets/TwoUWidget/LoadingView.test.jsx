import React from 'react';
import { shallow } from 'enzyme';

import LoadingView from './LoadingView';

describe('TwoUWidget LoadingView', () => {
  test('snapshot', () => {
    expect(shallow(<LoadingView />)).toMatchSnapshot();
  });
});
