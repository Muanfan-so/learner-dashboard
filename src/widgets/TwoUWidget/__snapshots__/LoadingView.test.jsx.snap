// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TwoUWidget LoadingView snapshot 1`] = `
<div
  className="loading-callouts-container"
>
  <h2
    className="skeleton-title-width"
  >
    <Component />
  </h2>
  <p>
    <Component />
  </p>
</div>
`;
